xml/admin_saltcluster.xml

```diff
@@ -1376,6 +1376,226 @@ OSD_ID  HOST         STATE                    PG_COUNT  REPLACE  FORCE  STARTED_
    </para>
   </note>
 
+  <sect2 xml:id="updating-ceph-https-proxy">
+   <title>Configuring HTTPS proxy</title>
+   <para>
+    &ceph; clusters are supposed to use an HTTPS proxy to communicate with the
+    container registry server. To configure the proxy, follow these steps on
+    the &adm;:
+   </para>
+   <procedure>
+    <step>
+     <para>
+      Copy the configuration file for containers:
+     </para>
+<screen>&prompt.sudo;cp /usr/share/containers/containers.conf /etc/containers/containers.conf</screen>
+    </step>
+    <step>
+     <para>
+      Edit the newly copied file and add the <option>http_proxy</option>
+      setting to its <literal>[engine]</literal> section, for example:
+     </para>
+<screen>&prompt.user;cat /etc/containers/containers.conf
+[engine]
+http_proxy=proxy.example.com
+[...]
+</screen>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="updating-ceph-local-registry">
+   <title>Creating the local registry (optional)</title>
```

Comment:
I think we have a guideline about avoiding the continuous tense in titles... If so, this should be:

Create the local registry (optional)

Reply:
Point me to the rule in the style guide please. We often use -ing form in titles

Reply:
OK, I think you are right. I cannot find it now. Comment withdrawn.